grain/BUILD

```diff
@@ -30,7 +30,10 @@ py_library(
         "sources.py",
         "transforms.py",
     ],
-    data = ["//grain/_src/python/experimental/index_shuffle/python:index_shuffle_module.so"],
+    data = select({
+        "@platforms//os:windows": ["//grain/_src/python/experimental/index_shuffle/python:index_shuffle_module.pyd"],
+        "//conditions:default": ["//grain/_src/python/experimental/index_shuffle/python:index_shuffle_module.so"],
+    }),
```

Comment:
Does Bazel allow to do it in a more concise manner? (Move path to a variable an glue the extension separately?)

Reply:
not sure, I think this is ok